addons/godot-next/references/array_2d.gd

```diff
@@ -229,24 +236,21 @@ func has(p_value) -> bool:
 	return false
 
 
-func invert() -> Array2D:
+func invert() -> void:
```

Comment:
I believe the intent for returning `self` in all of these calls was to be able to chain calls together. Is there a specific reason you think that is a bad design that should be changed? Or did you just do this because you don't think it should be statically returning the `Array2D` type?

Reply:
Mmm... that makes sense. The reason I removed the return was because just having the `Array2D` string there causes the object leak. I've been poking the code and looks like we can just return as the class we are extending and the object leak doesn't happen. Let me rework this a bit to stick to the original version a bit more.

Reply:
I've updated the PR. This is more in line with the original.